version.properties

```diff
@@ -1,6 +1,6 @@
 #Version of the produced binaries. This file is intended to be checked-in.
 #It will be automatically bumped by release automation.
-version=1.0.13
+version=2.0.0
```

Comment:
I wonder (rather more as a consideration than a real suggestion) if those are all of the backward incompatible changes that are planned by the way of bumping the major version number?

Reply:
Ok, reading the body of this PR I realized that it is taken into consideration :).